contactaccessrights/tests/phpunit/CRM/Contactaccessrights/Utils/ACLTest.php

```diff
@@ -1,116 +1,100 @@
-    $location1 = $this->createOptionValue('hrjc_location', 'Location 1');
-    $location2 = $this->createOptionValue('hrjc_location', 'Location 2');
-
-    // Add Regions / Locations to localAdmin's Rights
-    $this->setContactRights($localAdmin, $region1);
-    $this->setContactRights($localAdmin, $location1);
-    $this->setContactRights($localAdmin, $location2);
-
-    // Get Job Roles Clause
-    $aclUtil = new CRM_Contactaccessrights_Utils_ACL($localAdmin['id']);
-    $whereTables = $aclUtil->getWhereTables();
-    $jobsClause = $whereTables['car_3_jr'];
-
-    // Check Locations Clause
-    $locationMatches = [];
-    preg_match("/car_jr.location IN \(('[^']*'(, ?'[^']*')*)\)/", $jobsClause, $locationMatches);
-    $this->assertTrue(isset($locationMatches[1]), 'Jobs clause does not have expected format for locations.');
-
-    $locations = array_map('trim', explode(',', $locationMatches[1]));
-    $this->assertContains("'{$location1['value']}'", $locations, 'Expected value in clause for locations is not found.');
-    $this->assertContains("'{$location2['value']}'", $locations, 'Expected value in clause for locations is not found.');
-
-    // Check Regions Clause
-    $regionMatches = [];
-    preg_match("/car_jr.region IN \(('[^']*'(, ?'[^']*')*)\)/", $jobsClause, $regionMatches);
-    $this->assertTrue(isset($regionMatches[1]), 'Jobs clause does not have expected format for Regions.');
-
-    $regions = array_map('trim', explode(',', $regionMatches[1]));
-    $this->assertContains("'{$region1['value']}'", $regions);
-    $this->assertNotContains("'{$region2['value']}'", $regions);
-  }
-
-  /**
-   * Creates a new record for Rights entity for the given contact.
-   * 
-   * @param array $contact
-   *   Associative array with details for the contact
-   * @param array $entity
-   *   Details of either Region or Location for which the contact will be 
-   *   granted access
-   * @return array
-   *   Details of access right created in an associative array
-   */
-  private function setContactRights($contact, $entity) {
-    return RightsFabricator::fabricate([
-      'contact_id' => $contact['id'],
-      'entity_id' => $entity['id'],
-      'entity_type' => $entity['entity_type']
-    ]);
-  }
-
-  /**
-   * Creates a Record in Option Value for given Option Group
-   * 
-   * @param string $optionGroup
-   *   Name of the option group to which the option will be added
-   * @param string $value
-   *   Value for the new option value, used as its name, label and value
-   * @return array
-   *   Details of added option value + entity_type attribute
-   */
-  private function createOptionValue($optionGroup, $value) {
-    $params = [
-      'option_group_id' => $optionGroup,
-      'name' => $value,
-      'label' => $value,
-      'value' => $value
-    ];
-    $result = OptionValueFabricator::fabricate($params);
-    $result['entity_type'] = $optionGroup;
-    
-    return $result;
-  }
-
-}
+<?php
```

Comment:
I wonder why github is saying that this whole file was updated. Do you have any ideas?

Reply:
tbh, I was surprised when i saw such a huge change, i only made the tests class to extend BaseHeadless and I saw a lot of changes in the diff when making the PR

Reply:
Maybe when you edited the file the editor also updated the newline character? I can see that, on staging, this file uses CRLF while most of the other files use LF. What is it using after your change?